README.md

```diff
@@ -3,11 +3,19 @@ This repository provides a library of add-on custom operators for [ONNX Runtime]
 
 # Getting started
 Windows:
-- Install Visual Studio with C++ development tools
+- Install Visual Studio with C++ development tools, and prepare Python env if Python support needed.
```

Comment:
small grammar comment - on both this line and the one below for Linux, suggest to simplify to "and prepare Python environment as needed"